startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
I don't see how this is any better than what we're currently doing. Exiting here by killing bash will kill chaos [since it replaces itself with `startup.sh` when it needs an update](https://github.com/chaosbot/chaos/blob/139b8f1f825c040df9f4db1cd1285c15a2eb3910/cron/poll_pull_requests.py#L113-L114). Supervisor will make a few restart attempts, but since nothing will have changed, those will fail. What we really need is a way to test **each PR** before merging it.